### PR TITLE
DOCSP-30554: run a command

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-rust/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-30554>
[**Staging**](https://docs-mongodbcom-staging.corp.mongodb.com/rust/docsworker-xlarge/DOCSP-30554-run-command/fundamentals/run-command/) & [index](https://docs-mongodbcom-staging.corp.mongodb.com/rust/docsworker-xlarge/DOCSP-30554-run-command/#fundamentals), left hand nav

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?

### CODE REVIEW

File: source/fundamentals/run-command.txt [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+.. literalinclude:: /code-snippets/crud/run-command.rs
+   :language: rust
+   :dedent:
+   :start-after: start-runcommand
+   :end-before: end-runcommand
+
+Output
+~~~~~~
+
+In the output, you should see fields explaining the
```
Comment (mongoKart) [inline source/fundamentals/run-command.txt:133]: I: [should](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#s:~:text=the%20setup%20procedure.-,should,-Use%20should%20only)

```suggestion
The output includes fields explaining the
```

File: source/fundamentals/run-command.txt [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+   let result: Document = my_db.run_command(doc! { "hello": 1 }, None).await?;
+
+For a full list of database commands and corresponding parameters, see
+the :ref:`Additional Information section <addl-info-runcommand>`.
+
+.. note:: Read Preference
+
+   The ``run_command()`` method does not obey the read preference you
+   may have set on your ``Database`` object elsewhere in your code. By
```
[the remark this answers is not in the document]
Comment (rustagir) [reply source/fundamentals/run-command.txt:54]: interesting, thanks for linking!

File: source/fundamentals/run-command.txt [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+      );
+      let result: Document = my_db.run_command(command_doc, command_options).await?;
+   
+   For more information on read preference options, see :manual:`Read
+   Preference </core/read-preference/>` in the Server manual.
+
+Response
+--------
+
+Each method returns a ``Document`` object that contains
```
[the remark this answers is not in the document]
Comment (rustagir) [reply source/fundamentals/run-command.txt:75]: fixed

File: source/fundamentals/run-command.txt [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+the response from the database after the command has been executed. Each
+database command performs a different function, so the response content
+can vary across commands. However, every response contains documents
```
[the remark this answers is not in the document]
Comment (rustagir) [reply source/fundamentals/run-command.txt:78]: just one

File: source/fundamentals/run-command.txt [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+   :header-rows: 1
+   :widths: 30 70
+
+   * - Field
+     - Description
+
+   * - <command result>
+     - Provides fields specific to the database command. For example,
+       ``count`` returns the ``n`` field and ``explain`` returns the
+       ``queryPlanner`` field.
```
[the remark this answers is not in the document]
Comment (rustagir) [reply source/fundamentals/run-command.txt:91]: agreed, keeping the verb for consistency with other entries.

File: source/fundamentals/run-command.txt [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+     - Indicates the logical time of the operation. MongoDB uses the
+       logical time to order operations.
+       
+       .. seealso::
+  
+          To learn more about logical time, see our blog post about the
+          :website:`Global Logical Clock </blog/post/transactions-background-part-4-the-global-logical-clock>`.
+
+   * - ``$clusterTime``
+     - Provides a document that returns the signed cluster time. Cluster time is a
```
[the remark this answers is not in the document]
Comment (rustagir) [reply source/fundamentals/run-command.txt:107]: see above

**Review by mongoKart — APPROVED** [review]
lgtm

File: source/code-snippets/crud/run-command.rs [hunk 7]
```diff
@@ -0,0 +1,24 @@
+use mongodb::{ bson::{ doc, Document }, Client, Database };
+
+#[tokio::main]
+async fn main() -> mongodb::error::Result<()> {
+    let uri: &str = "<connection string>";
```
Comment (abr-egn) [inline source/code-snippets/crud/run-command.rs:5]: None of the type annotations for `let` bindings in this function are needed.

**Review by abr-egn — COMMENTED** [review]
LGTM with one minor comment!

**Pull request merged**